subprojects/core/src/integTest/groovy/org/gradle/api/resource/TextResourceIntegrationTest.groovy

```diff
@@ -114,7 +114,7 @@ class TextResourceIntegrationTest extends AbstractIntegrationSpec {
 
         buildFile << """
             task uriText(type: MyTask) {
-                config = resources.text.fromUri("http://localhost:$server.port/myConfig-${uuid}.txt")
+                config = resources.text.fromUri("$server.uri/myConfig-${uuid}.txt")
```

Comment:
minor: you've used braces for non-trivial GString symbols in other places

Reply:
Good spot. I'll fix this.